src/lib/controllers/environment-file/environment.ts

```diff
@@ -0,0 +1,30 @@
+import {
+	DockerDeployTarget,
+	EnvConfigStore,
+	EnvironmentObject,
+	KubernetesConfigStore,
+	KubernetesDeployTarget,
+} from '.';
+
+export class Environment implements EnvironmentObject {
```

Comment:
This only seems exist only to provide a data structure for the EnvironmentObject, and doesn't actually offer any class based benefits. Why instantiate as a class instead of just creating a new interface object where it's used?

This would also have the effect of not having a class where all it's members are publicly exposed, which kind of defeats the point.

Reply:
Good point. removed.